roles/dns/tasks/main.yml

```diff
@@ -1,7 +1,10 @@
 ---
 - name: Install Bind
-  package: name=bind state=present
   when: not openshift.common.is_containerized | bool
+  package: name=bind state=present
+  register: l_result
+  retries: 4
+  until: l_result.rc | default(-1) == 0
```

Comment:
I prefer this. Unless absolutely needed I don't think we should parse the result string.